impl/asm/src/main/java/org/wildfly/transformer/asm/ReflectionModel.java

```diff
@@ -8,18 +8,15 @@
  *
  * @author Scott Marlow
  */
-class BataviaReflectionModel {
+class ReflectionModel {
```

Comment:
ReflectionModel utility class should have final qualifier.

Reply:
+1, changed.